src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1385,7 +1384,7 @@ internal void SaveAs(string path)
         {
             try
             {
-                Model.CurrentWorkspace.SaveAs(path, EngineController.LiveRunnerRuntimeCore);
+                model.SaveAs(path);
```

Comment:
thinking about this more - I agree with @ramramps and think we should **not** add another API for custom node saving - as potentially that won't exist in 2.0.